5001-6000/LIT5673AkkonuColl.xml

```diff
@@ -209,10 +212,28 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 </div>
 
                 <div type="textpart" xml:id="AkkColl6">
-                    <label>Miracle of the monk Isaac</label>
-
-                    <div type="textpart" xml:id="AkkMir6" corresp="LIT3592Miracle">
-                        <label>Miracle</label>
+                    <label>Miracle of the monk Yǝsḥaq</label>
+                    <note> Two variant versions of this miracle, with the same recipient but
+                        significant thematic differences, exist. To avoid confusion and overlap, the
+                        miracle has been subdivided into two distinct variants. </note>
+
+                    <div type="textpart" xml:id="AkkMir6A" corresp="LIT3592MiracleA">
+                        <label>The Revelation of the Virgin Mary to the Devoted Monk Yǝsḥaq</label>
+                        <note> Yǝsḥaq, a monk under Bishop Gabriel of Alexandria, prayed for seven
```

Comment:
these notes should be removed here and become abstracts under respective work records LIT3592Miracle for the first miracle and the new ID for the second one